src/functions/anti-raid/checkBan.ts

```diff
@@ -0,0 +1,21 @@
+import { GuildMember, PermissionFlagsBits } from 'discord.js';
+
+export function checkBan(member: GuildMember, authorId: string, modRoleId: string): string | null {
```

Comment:
I feel like this would be better used as an enum or type rather than the overly generic "string"